scripts/check-env.js

```
@@ -0,0 +1,245 @@
+
+/**
+ * Script to check environment variables before starting the application
+ * This script is meant to be run as part of the build/start process
+ * 
+ * This script dynamically detects environment variables and categorizes them
+ * based on prefixes and common patterns.
+ */
+
+// Load environment variables from .env files
+require('dotenv').config({ path: '.env.local' });
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)